lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_PositionStatePMPS.TcPOU

```diff
@@ -12,9 +12,16 @@ VAR_IN_OUT
     fbArbiter: FB_Arbiter;
     fbFFHWO: FB_HardwareFFOutput;
 END_VAR
+VAR_INPUT
+    stFullBeam: ST_BeamParams := PMPS_GVL.cstFullBeam;
```

Comment:
can we call it like Maximum Allowed Beam? Just so we don't confuse it with FullBeam.

Reply:
I agree with a rename, but that name is not quite right either. We're actually defining a minimum beam setting at which a state is configured safe enough for the FFO to be automatically reset, not a maximum allowed beam.

Reply:
PossibleMaxBeam? MaxBeamPossible?

Reply:
`stMaxBeamThreshold` is what I'm going with